derive_builder/Cargo.toml

```diff
@@ -23,6 +23,7 @@ proc-macro = true
 
 [features]
 logging = [ "log", "env_logger", "derive_builder_core/logging" ]
+try_from = []
 struct_default = []
 skeptic_tests = ["skeptic"]
 nightlytests = []
```

Comment:
let's change this to `nightlytests = ["try_from"]` - this will in turn activate it in travis and `dev/nightlytests.sh` / `dev/githook.sh`.